slasher/detection/listeners.go

```diff
@@ -69,3 +76,20 @@ func (ds *Service) detectIncomingAttestations(ctx context.Context, ch chan *ethp
 		}
 	}
 }
+
+func extractSignedBeaconBlockHeader(block *ethpb.SignedBeaconBlock) (*ethpb.SignedBeaconBlockHeader, error) {
```

Comment:
this says `extractSignedBeaconBlockHeader` but the function looks like you are extracting signing root of block body?